xdsl/dialects/onnx.py

```diff
@@ -255,6 +255,83 @@ def verify_(self) -> None:
             )
 
 
+@irdl_op_definition
+class Reshape(IRDLOperation):
+    name = "onnx.Reshape"
+    T = Annotated[AnyFloat | IntegerType, ConstraintVar("T")]
+    data = operand_def(TensorType[T])
+    shape = operand_def(TensorType[IntegerType(64)])
```

Comment:
I have a feeling we can't specify the bitwidth here, will have to do that by hand in the `verify_` method

Reply:
yeah, I think so, @kayode-gif a usage of the `width` field can be found here: https://github.com/xdslproject/xdsl/blob/fbc1a7996ff6e433e9a9b24a679b7165d00b7142/xdsl/dialects/builtin.py#L425
and an example of a custom `verify_` method here: https://github.com/xdslproject/xdsl/blob/fbc1a7996ff6e433e9a9b24a679b7165d00b7142/xdsl/dialects/seq.py#L58

Reply:
i've added that verification check in the `verify_` method now, i don't think i have to do any other check with the bitwidth?

Reply:
@kayode-gif I've added this in my initial comments below